### ISSUE TITLE
Fix jsdom errors when running unit test

### ISSUE THREAD

**Issue by Cooryd** [libero/reviewer-client#69]
run `yarn test`

This is a regression see: libero/reviewer#367

```console.error node_modules/jsdom/lib/jsdom/virtual-console.js:29
2019-12-16T15:06:29.0420090Z       Error: Error: connect ECONNREFUSED 127.0.0.1:80
2019-12-16T15:06:29.0420654Z           at Object.dispatchError (/home/runner/work/reviewer-client/reviewer-client/node_modules/jsdom/lib/jsdom/living/xhr-utils.js:65:19)
2019-12-16T15:06:29.0421302Z           at Request.<anonymous> (/home/runner/work/reviewer-client/reviewer-client/node_modules/jsdom/lib/jsdom/living/xmlhttprequest.js:676:20)
2019-12-16T15:06:29.0421724Z           at Request.emit (events.js:215:7)
2019-12-16T15:06:29.0422331Z           at Request.onRequestError (/home/runner/work/reviewer-client/reviewer-client/node_modules/request/request.js:881:8)
2019-12-16T15:06:29.0422859Z           at ClientRequest.emit (events.js:210:5)
2019-12-16T15:06:29.0423219Z           at Socket.socketErrorListener (_http_client.js:406:9)
2019-12-16T15:06:29.0423524Z           at Socket.emit (events.js:210:5)
2019-12-16T15:06:29.0423803Z           at emitErrorNT (internal/streams/destroy.js:92:8)
2019-12-16T15:06:29.0424118Z           at emitErrorAndCloseNT (internal/streams/destroy.js:60:3)
2019-12-16T15:06:29.0424593Z           at processTicksAndRejections (internal/process/task_queues.js:80:21) undefined
```

**Comment by Cooryd** [comment]
After well over a week of investigating, we've not been able to get to the bottom of this. 
An incomplete list of what we've tried (some wasn't noted down):
- Removed all context providers and their component functionality (Apollo mutations/ query, router links) still throws the error. Leads us to believe the Apollo mock provider isn't the issue.
- Made the component take no props and just output the two tests expected strings works but obviously isn't a suitable fix.
- Removed all use of imported components. No luck here.
- Refactored the use of `findByText` to use flush promises and use `container.querySelector` we then seem to get an unwrapped `act` error instead. not obvious where or why this is thrown either.

We guess this issue may be caused by one of the libraries being used (jest? testing-library?) but again we can't be sure. We've decided to put this back into the backlog for now to see if a fix comes out of a dependency update. If not we're hoping fresh eyes on this may spot something we haven't yet. We've decided to add in a temp suppression and warning re-throw #72 for now to stop anyone else going down the rabbit hole. The errors thrown weren't affecting the tests reliability and the unit tests are still behaving as expected (with pass/ fail) so this seemed preferable over removing the tests for now.
